libpod/runtime.go

```diff
@@ -947,6 +947,21 @@ func (r *Runtime) generateName() (string, error) {
 	// The code should never reach here.
 }
 
+//CheckName makes sure a container or pod does not exist with the given name
+func (r *Runtime) CheckName(name string) error {
```

Comment:
Shouldn't this return a bool, not an error?

Reply:
sure, that's probably easier to understand